security/pkg/nodeagent/sds/server.go

```diff
@@ -163,6 +181,38 @@ func NewPlugins(in []string) []plugin.Plugin {
 	return plugins
 }
 
+func (s *Server) initDebugServer(port int) {
+	mux := http.NewServeMux()
+	mux.HandleFunc(fmt.Sprintf("%s/sds/workload", debugBase), s.workloadSds.debugHTTPHandler)
+	mux.HandleFunc(fmt.Sprintf("%s/sds/gateway", debugBase), s.gatewaySds.debugHTTPHandler)
+	s.debugServer = &http.Server{
+		Addr:    fmt.Sprintf("127.0.0.1:%d", port),
+		Handler: mux,
+	}
+
+	go func() {
+		err := s.debugServer.ListenAndServe()
+		log.Errorf("debug server failure: %s", err)
+	}()
+}
+
+func (s *sdsservice) debugHTTPHandler(w http.ResponseWriter, req *http.Request) {
+	w.Header().Add("Content-Type", "application/json")
+
+	workloadJSON, err := s.DebugInfo()
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		failureMessage := fmt.Sprintf("debug endpoint failure: %s", err)
+		if _, err := w.Write([]byte(failureMessage)); err != nil {
+			log.Errorf("debug endpoint failed to write error response: %s", err)
+		}
+		return
+	}
+	if _, err := w.Write([]byte(workloadJSON)); err != nil {
+		log.Errorf("debug endpoint failed to write response: %s", err)
```

Comment:
Could you replace log.Errorf to sdsServiceLog.Errorf and use sdsServiceLog for other logs?